client/components/Footer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+
+class Footer extends Component {
+  render() {
+    return (
+    <div className="container-fluid">
+      <div className="footer">
+        <div>
+          <p><i>Designed with</i> <i className="fa fa-heart love" aria-hidden="true"></i> <i>by </i><strong>Ucheya</strong></p>
```

Comment:
Lines of code should not exceed 80 characters

Reply:
Done, except the code lines not flagged by eslint